### ISSUE TITLE
Setup Travis CI build so asdf plugin-test can validate the plugin (and so it can be listed in the asdf README)

### ISSUE THREAD

**Issue by Stratus3D** [sylph01/asdf-mongodb#1]
I'd like to list this plugin in the asdf readme, but the plugin must pass asdf's plugin-test command first. Can you setup a travis CI build for the repository? It's really easy to do. Here is the .travis.yml file from my asdf-lua plugin: https://github.com/Stratus3D/asdf-lua/blob/master/.travis.yml

**Comment by sylph01** [comment]
Thank you for your advice (and sorry for the late response...)!
I have added the `.travis.yml` file, and in the process, I found that it wasn't working, so I also fixed the `list-all`. The commits are as follows:
- https://github.com/sylph01/asdf-mongodb/commit/db42427c1baf065c2554839b20135aeee67e884a
- https://github.com/sylph01/asdf-mongodb/commit/14968a85fc3dc73df3d7b1c255d6d88f3d937da0

Now that I know how to setup Travis CI for plugins, I will add it to my other plugins. Thank you very much! 👍
